src/server/scripts/ExilesReach/zone_exiles_reach.cpp

```diff
@@ -5920,6 +5925,886 @@ class quest_resizing_the_situation_horde : public quest_resizing_the_situation
+    {
+        switch (spellInfo->Id)
+        {
+            case SPELL_PING_GARRICK_TO_RIDE_BOAR:
+            {
+                Unit* owner = me->GetOwner();
+                if (!owner)
+                    break;
+
+                PhasingHandler::InheritPhaseShift(me, owner);
+                PhasingHandler::ResetPhaseShift(me);
+
+                _events.ScheduleEvent(EVENT_CAPTAIN_GARRICK_RIDE_BOAR, 2s);
+                break;
+            }
+            case SPELL_ENHANCED_BOAR_PING_VEHICLE:
+            {
+                PhasingHandler::AddPhase(me, PHASE_SEE_TORGOK, true);
+                _events.ScheduleEvent(EVENT_CAPTAIN_GARRICK_RIDE_BOAR_EXIT_BOAR_CONVERSATION, 500ms);
+                break;
+            }
+            case SPELL_PING_GARRICK_TORGOK:
+            {
+                Creature* henry = FindCreatureIgnorePhase(me, "henry_garrick_ogre_ruins_prisoner");
+                if (!henry)
+                    break;
+
+                Player* player = me->GetOwner()->ToPlayer();
+                if (!player)
+                    break;
+
+                Creature* henryPersonal = henry->SummonPersonalClone(henry->GetPosition(), TEMPSUMMON_MANUAL_DESPAWN, 0s, 0, 0, player);
+                if (!henryPersonal)
+                    break;
+
+                me->SetReactState(REACT_AGGRESSIVE);
+                _henryGUID = henryPersonal->GetGUID();
+                _events.ScheduleEvent(EVENT_CAPTAIN_GARRICK_RIDE_BOAR_TALK_TO_HENRY, 1s);
+                break;
+            }
+            default:
+                break;
+        }
+    }
+
+    void UpdateAI(uint32 diff) override
+    {
+        _events.Update(diff);
+
+        while (uint32 eventId = _events.ExecuteEvent())
+        {
+            switch (eventId)
+            {
+                case EVENT_CAPTAIN_GARRICK_RIDE_BOAR:
+                {
+                    if (Unit* owner = me->GetOwner())
+                    {
+                        Creature* boar = owner->FindNearestCreatureWithOptions(10.0f, { .CreatureId = NPC_GIANT_BOAR, .OwnerGuid = owner->GetGUID() });
+                        if (!boar)
+                            return;
+
+                        boar->SetTemplateRooted(false);
+                        me->CastSpell(boar, SPELL_RIDE_VEHICLE_CAPTIAN_BOAR);
+                    }
+                    break;
+                }
+                case EVENT_CAPTAIN_GARRICK_RIDE_BOAR_CHECK_OWNER:
+                {
+                    if (Unit* owner = me->GetOwner())
+                    {
+                        if (Player* player = owner->ToPlayer())
+                        {
+                            if (player->GetQuestStatus(QUEST_RIDE_ENHANCED_BOAR) == QUEST_STATUS_INCOMPLETE)
+                            {
+                                if (player->IsInCombat() && !me->IsInCombat())
+                                    me->Attack(player->GetVictim(), true);
```

Comment:
AI()->AttackStart() directly sets chase movement, which is something we might want here
```suggestion
                                    me->AI()->AttackStart(player->GetVictim());
```